CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Unreleased changes
 
+- Add an option `--grpc2-health-min-peers` (environment variable `CONCORDIUM_NODE_GRPC2_HEALTH_MIN_PEERS`)
```

Comment:
Couldn't it just always be checking whether the `desired_peers` is fulfilled? As I recall, then the node will keep trying get more peers until the number of desired peers is fulfilled.  

What will the concrete value be for such an option as compared to `desired_peers`? Will it be lower? And if so why? If a node is not able to get its `desired` peers should it then be considered healthy with respect to its settings?

Reply:
I think using desired peers is too limiting. Just because the node does not have the desired number of peers does not make it unhealthy and is not cause for alerts for example.

This bound should be "minimum required" and in practice for startup we sometimes need to set minimum lower than  desired.

Reply:
Having this option does still give the operator an option to set it to the same value as desired peers. But there are use-cases where they should be different.

Reply:
I would personally refrain from using the desired peers since a node can be healthy and operating before having all desired peers. Maybe it could make sense to have the desired peers as default if the environment variable or option is not defined.

Reply:
Okay, I was just wondering if this would ever be lower than 'desired'. But if that is the case then lgtm.

Reply:
Well.. I presume the node can be "healthy and operating" also when lower than "minimum peers" unless you always set this to 1.

Reply:
It's as if there is ambiguity and usage dependent scenarios that we have to deal with :) 

Thanks for the feedback.